modules/evaluate_condition_abstract.py

```diff
@@ -0,0 +1,21 @@
+"""
+Abstract base class for evaluating drone conditions
+"""
+import abc
+
+
+class EvaluateCondition(abc.ABC):
```

Comment:
Nit: I was thinking about naming after our discussion. I think naming the base class `Condition` would be good.